yaml/generator.go

```diff
@@ -78,6 +78,16 @@ func (c *TestCase) generateDockerComposeFile() []byte {
 	vars["LokiHTTPPort"] = c.PortConfig.LokiHTTPPort
 	vars["TempoHTTPPort"] = c.PortConfig.TempoHTTPPort
 
+	env := []string{}
+
+	for k, v := range vars {
+		env = append(env, "OATS_"+strings.ToUpper(k)+"="+v.(string))
+	}
+
+	for _, v := range c.Definition.DockerCompose.Environment {
+		env = append(env, "OATS_"+v)
```

Comment:
```suggestion
		env = append(env, k+"="+v)
```

Reply:
we also may want to pass in environment variables to the application directly, so prepending OATS_ won't work.